webapp/channels/src/components/channel_info_rhs/header.tsx

```diff
@@ -89,9 +88,9 @@ const Header = ({channel, isArchived, isMobile, onClose}: Props) => {
                     aria-label='Close'
                     onClick={onClose}
                 >
-                    <LocalizedIcon
+                    <i
                         className='icon icon-close'
-                        ariaLabel={{id: t('rhs_header.closeTooltip.icon'), defaultMessage: 'Close Sidebar Icon'}}
+                        aria-label={formatMessage({id: 'rhs_header.closeTooltip.icon', defaultMessage: 'Close Sidebar Icon'})}
```

Comment:
actually could you add the aria-label to the parent button and remove from here, because a button is a11y focusable not the icon inside of it. icon can just be classname

Reply:
@M-ZubairAhmed, Sure, I will do that.